<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Georgia</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Gami-ccount Launcher</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ui/gami-ccount_icon.png</normaloff>ui/gami-ccount_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(53, 53, 53);</string>
  </property>
  <widget class="QLabel" name="lbl_pw">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>350</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 18px;
color: 'white';</string>
   </property>
   <property name="text">
    <string>PASSWORD</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailfield">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: 'white';</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_createaccount">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>630</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:'white';
color:'black';
font-weight:1200; 
border-radius: 18px;</string>
   </property>
   <property name="text">
    <string>CREATE ACCOUNT</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_registration">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ink Free</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:'white';</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;USER&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_email">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 18px;
color: 'white';</string>
   </property>
   <property name="text">
    <string>EMAIL</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordfield">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>340</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: 'white';</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmpasswordfield">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>400</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: 'white';</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_confirm_pw">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 18px;
color: 'white';</string>
   </property>
   <property name="text">
    <string>CONFIRM PASSWORD</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_invalid_msg">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>460</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font-size: 20px;</string>
   </property>
   <property name="text">
    <string>Invalid Email or Password!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>630</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(0, 170, 0);
font-weight: bold;
color:'yellow';
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_ccount2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>301</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 'white';
color: 'white';
font-size: 75px;
font-weight: bold;</string>
   </property>
   <property name="text">
    <string>-ccount</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_gami2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>291</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 'white';
color: rgb(170, 0, 255);
font-size: 75px;;
font-weight: bold;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;Gami&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_registration_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ink Free</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;REGISTRATION&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <zorder>lbl_pw</zorder>
  <zorder>emailfield</zorder>
  <zorder>btn_createaccount</zorder>
  <zorder>lbl_registration</zorder>
  <zorder>lbl_email</zorder>
  <zorder>passwordfield</zorder>
  <zorder>confirmpasswordfield</zorder>
  <zorder>lbl_confirm_pw</zorder>
  <zorder>lbl_invalid_msg</zorder>
  <zorder>btn_back</zorder>
  <zorder>lbl_gami2</zorder>
  <zorder>lbl_registration_2</zorder>
  <zorder>lbl_ccount2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
